en/middleware/micrortps_manual_code_generation.md

```diff
@@ -1,6 +1,6 @@
 # Manually Generate Client and Agent Code
 
-This topic shows how to manually generate the code for the client and the agent (instead of [automatically generating](../middleware/micrortps.md) it when the PX4 Firmware is compiled). 
+This topic shows how to manually generate the code for the client and the agent (instead of [automatically generating](../middleware/micrortps.md) it when the PX4 firmware is compiled).
```

Comment:
@TSC21 I would add a note below here explaining when/why you would want to do this.

If this is just an option, in the note you might say that, and recommend that they follow the "normal" instructions.

Reply:
Fair enough

Reply:
OK, leaving that with you. If you do it by tomorrow I can probably merge then.